packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -182,21 +185,25 @@ const Updates = decorate([
       toggleState,
       update: () => xoaUpdater.update(),
       upgrade: () => xoaUpdater.upgrade(),
-      onChannelChange: (_, channel) => {
-        if (channel.value === UNLISTED_CHANNEL_VALUE) {
-          return {
-            channel: channel.value,
-            addUnlistedChannel: true,
-          }
-        }
-        return { channel: channel.value, addUnlistedChannel: false }
-      },
     },
     computed: {
       areJobsRunning: (_, { jobs, backupNgJobs }) =>
         jobs !== undefined &&
         backupNgJobs !== undefined &&
         some(jobs.concat(backupNgJobs), job => job.runId !== undefined),
+      channelsFormId: generateId,
+      channelsOptions: ({ channels }) => [
+        ...Object.keys(channels)
```

Comment:
When `channels` is `undefined`, this computed throws 
`factory.js:199 computed "channelsOptions" thrown TypeError: Cannot convert undefined or null to object`
Possible correction `Object.keys(channels || {})`